doc/source/monitor-simulation.rst

```diff
@@ -0,0 +1,230 @@
+
+Open the respective configuration files and change them. Depending on your device, use one of the two following commands:
+
+.. code-block:: bash
+
+  # M1 macOS
+  open /opt/homebrew/etc/prometheus.yml
+
+  # Intel macOS
+  open /usr/local/etc/prometheus.yml
+
+
+and then delete all the text in the file and paste a new Prometheus config you see below. You may adjust the time intervals to your requirements:
+
+.. code-block:: bash
+
+  global:
+    scrape_interval: 1s
+    evaluation_interval: 1s
+
+  scrape_configs:
+  # Scrape from each ray node as defined in the service_discovery.json provided by ray.
+  - job_name: 'ray'
+    file_sd_configs:
+    - files:
+      - '/tmp/ray/prom_metrics_service_discovery.json'
+
+Now after you have edited the Prometheus configuration, do the same with the Grafana configuration files. Open those using one of the following commands as before:
+
+.. code-block::python
+  # M1 macOS
+  open /opt/homebrew/etc/grafana/grafana.ini
+
+  # Intel macOS
+  open /usr/local/etc/grafana/grafana.ini
+
+
+Your terminal editor should open and allow you to apply the following configuration as before.
+
+.. code-block:: bash
+
+  [security]
+  allow_embedding = true
+
+  [auth.anonymous]
+  enabled = true
+  org_name = Main Org.
+  org_role = Viewer
+
+  [paths]
+  provisioning = /tmp/ray/session_latest/metrics/grafana/provisioning
+
+Congratulations, you just downloaded all the necessary software needed for metrics tracking. Now, let’s start it.
+
+Tracking metrics
+----------------
+
+Before running your Flower simulation, you have to start the monitoring tools you have just installed and configured.
+
+.. code-block:: bash
+
+  brew services start prometheus
+  brew services start grafana
+
+Please include the following argument in your Python code when starting a simulation.
+
+.. code-block:: python
+
+  fl.simulation.start_simulation(
+      # ...
+      # all the args you used before
+      # ...
+      ray_init_args = {"include_dashboard": True}
+  )
+
+Now, you are ready to start your workload.
+
+Shortly after the simulation starts, you should see the following logs in your terminal:
+
+.. code-block:: bash
+
+  2023-01-20 16:22:58,620	INFO [worker.py:1529](http://worker.py:1529/) -- Started a local Ray instance. View the dashboard at http://127.0.0.1:8265
+
+
+You can look at everything at http://127.0.0.1:8265 .
+
+It's a Ray Dashboard. You can navigate to Metrics (on the left panel, the lowest option).
+
+Or alternatively, you can just see them in Grafana by clicking on the right-up corner, “View in Grafana”. Please note that the Ray dashboard is only accessible during the simulation. After the simulation ends, you can only use Grafana to explore the metrics. You can start Grafana by going to http://localhost:3000/.
+
+After you finish the visualization, stop Prometheus and Grafana. This is important as they will otherwise block, e.g. port 3000 on your machine as long as they are running.
```

Comment:
```suggestion
After you finish the visualization, stop Prometheus and Grafana. This is important as they will otherwise block, for example, port ``3000`` on your machine as long as they are running.
```